test/conftest.py

```diff
@@ -961,6 +961,9 @@ def get_document_store(
             similarity=similarity,
         )
 
+    elif document_store_type == "memory_bm25":
+        document_store = InMemoryDocumentStore(index=index, use_bm25=True)
+
```

Comment:
I think we don't need this distinction, the normal `InMemoryDocumentStore` can always have `use_bm25=True`. Even if it impacts performance a bit i think we can take it. 

The risk with this arrangement is that some tests might not run in CI right now (yep that's on us, but let's not take the risk for now :sweat_smile: )